bundles/org.openhab.binding.enigma2/src/main/java/org/openhab/binding/enigma2/handler/Enigma2Handler.java

```diff
@@ -78,16 +92,20 @@ private void refresh() {
                 updateStatus(ThingStatus.OFFLINE);
             }
         } catch (Exception e) {
```

Comment:
I think i mentioned this elsewhere in the code already, so i might have overlooked this one. Please catch a specific exception instead.

Reply:
Fixed, the Exception was already caught in Enigma2Client#transmit() so it never coulf appear here